config/config.exs

```diff
@@ -7,6 +7,17 @@
 # General application configuration
 import Config
 
+config :screenplay, Screenplay.Repo,
+  database: "screenplay_dev",
+  username: System.get_env("DATABASE_USER", ""),
+  password: System.get_env("DATABASE_PASSWORD", ""),
+  hostname: System.get_env("DATABASE_HOST", "localhost"),
+  port: System.get_env("DATABASE_PORT", "5432") |> String.to_integer(),
+  show_sensitive_data_on_connection_error: true,
+  backoff_min: 5_000
+
+config :screenplay, ecto_repos: [Screenplay.Repo]
```

Comment:
Just curious, what does this do? I'm not seeing it used anywhere else, but maybe Ecto needs it?

Reply:
Yeah, it's just a piece of configuration that Ecto needs, according to the documentation

> There's one final bit of configuration that we'll need to add ourselves, since the generator does not add it. Underneath the configuration in config/config.exs, add this line:
>
> `config :friends, ecto_repos: [Friends.Repo]`
>
> This tells our application about the repo, which will allow us to run commands such as [mix ecto.create](https://hexdocs.pm/ecto/Mix.Tasks.Ecto.Create.html) very soon.